### ISSUE TITLE
Add Custom Animation Facility 

### ISSUE THREAD

**Issue by hussainahmad** [pavlospt/litho-glide#2]
Hey.. @pavlospt 
As Glide support custom animation features but I didn't find any solution how to use custom animation using resource file or custom class in `Litho-glide` . Is there any solution ?

**Comment by pavlospt** [comment]
@hussainahmad can you please point me to the relevant Glide documentation?

**Comment by hussainahmad** [comment]
@pavlospt thanks for responding , [Custom Animations with animate()](https://futurestud.io/tutorials/glide-custom-animations-with-animate) , I want to use `fadeIn` animation in my project , but`Litho-glide` support `crossFade` only animation, I think so if I am not wrong

**Comment by pavlospt** [comment]
@hussainahmad yes this is not supported. Will try to add it and update the issue :)
